packages/cli/src/controller/build-controller.ts

```diff
@@ -74,15 +74,15 @@ export async function runWebpack(
 
   await new Promise((resolve, reject) => {
     webpack(config).run((error, stats) => {
-      if (error) {
+      if (error || !stats) {
```

Comment:
I think this needs to be separated. If !stats then error could also be undefined so it would reject with undefined.